ambry-router/src/test/java/com/github/ambry/router/UndeleteManagerTest.java

```diff
@@ -363,8 +365,7 @@ private Properties getNonBlockingRouterProperties(int parallelism) {
 
   private void assertDeleted(String blobId) throws Exception {
     try {
-      router.getBlob(blobId, new GetBlobOptionsBuilder().build())
-          .get(AWAIT_TIMEOUT_MS, TimeUnit.MILLISECONDS);
+      router.getBlob(blobId, new GetBlobOptionsBuilder().build()).get(AWAIT_TIMEOUT_MS, TimeUnit.MILLISECONDS);
```

Comment:
rm formatting edits. 
PR can be smaller if all formatting edits are moved to a different PR.